sdk/keyvault/azure-security-keyvault-common/inc/azure/keyvault/common/client_options.hpp

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+/**
+ * @file
+ * @brief Defines the based options to create a Key Vault client.
+ *
+ */
+
+#pragma once
+
+#include <azure/core/http/http.hpp>
+#include <azure/core/internal/client_options.hpp>
+#include <azure/core/response.hpp>
+
+namespace Azure { namespace Security { namespace KeyVault { namespace Common {
```

Comment:
When do we plan to update the CL for KeyVault?

Reply:
After this PR and as part of the Readme updates before first preview